### ISSUE TITLE
Missing class

### ISSUE THREAD

**Issue by andref94** [usnistgov/tesim_omnetpp#2]
I've imported in OMNeT++ inet and MiXiM's project found in your zip but when I try to build MiXiM the compiler says that there are 8 errors in ClientNet.cc file. How can I fix the problem?
![screenshot](https://user-images.githubusercontent.com/26082760/46204868-04fea580-c31f-11e8-8da3-a3b17844eca5.jpg)

**Comment by ykLIU1982** [comment]
Hello, please double check two things here:
1) Please download the right version of OMNET++ as the newer versions have many changes;
2) Where are these errors? The major TESim project is in MiXiM/examples/BaseNetwork. If the errors are in the other MiXiM examples, you can just delete the folders.

**Comment by andref94** [comment]
Thank you for your suggestions.
I've deleted the other MiXiM folders, but now there's a new error: Cannot run program "/home/a/omnetpp-4.4.1/samples/MiXiM/tests/miximtests" (in directory "/home/a/omnetpp-4.4.1/samples/MiXiM/examples/baseNetwork"): error=2, File or directory not exists. Do I have to change the omnetpp.ini file?

**Comment by ykLIU1982** [comment]
Hi,

If you'd like run an example, just start the simulation from the omnetpp.ini in that example project. For example, to start the TESim example, go to MiXiM->examples->baseNetwork, right click the omnetpp.ini and select "Run as/Omnet++ Simulation". If you'd like to create your own OMNET++ project and load the TESim as a reference, please check the OMNET++ user manual and its online technical resources.

**Comment by andref94** [comment]
Hello,
if I import inet and MiXiM projects into OMNeT++ from your folder and I follow your instructions for running TESim example, I get the same error I wrote in my last comment. It seems like something is missing

**Comment by ykLIU1982** [comment]
Hi,
Please try to run the other examples to see if the OMNET++/INET/MiXiM is configured correctly.

**Comment by ykLIU1982** [comment]
Hi,
I double checked the previous problem you commented. The TESim example/project code is maintained in MiXiM/examples/baseNetwork folder along with function modules in INET/MiXiM/src folders. You can try the examples in the MiXiM/examples. We don't maintain the other project code, like the one you mentioned in your previous comment which is located in MiXiM/tests/miximtests.

**Comment by andref94** [comment]
Hi,
I've followed this steps: go to MiXiM->examples->baseNetwork, right click the omnetpp.ini and select "Run as/Omnet++ Simulation", but I've got again the same error.

Exception occurred executing command line.
Cannot run program "/home/a/omnetpp-4.4.1/samples/MiXiM/examples/miximexamples" (in directory "/home/a/omnetpp-4.4.1/samples/MiXiM/examples/CSMAMac"): error=2, File or directory not exists

All INET examples run correctly, instead all the examples in MiXiM/examples folder raise the same error, both baseNetwork both other folders

**Comment by ykLIU1982** [comment]
Hi,

I just tried to reinstall OMNET++, INET, and MiXiM in a Windows OS system. Following the install guide of OMNET++ and the installation steps of TESim, the example can work well. I notices that the import instructions for the MiXiM was directly copied from the ones for the INET which was a cross-reference typo. I just corrected it and updated the installation guide in the repo.

It seems that the earlier error is because that MiXiM was not imported. One way to solve it is just to delete the MiXiM project (right click on the whole MiXiM folder in the project explorer and select delete) and import it again. When importing the MiXiM, please check the MiXiM option in the same way as checking INET. INET and MiXiM should be built after importing respectively.

**Comment by andref94** [comment]
Hi,
I've noticed that the instructions for building MiXiM were uncorrect, but I've thought that I had to build also MiXiM's project. In your last answer I red that you're working on a Windows OS while I'm working on Ubuntu. I 've tried to import the project in a Windows OS and now I'm able to run all the examples in baseNetwork folder correctly. In the end, I think that the problem was due to the different OS. 
Thank you for your patience!
